conf/app.routes

```diff
@@ -49,3 +49,12 @@ POST       /ask-secondary-warehouses                        controllers.AskSecon
 GET        /change-ask-secondary-warehouses                  controllers.AskSecondaryWarehousesController.onPageLoad(mode: Mode = CheckMode)
 POST       /change-ask-secondary-warehouses                  controllers.AskSecondaryWarehousesController.onSubmit(mode: Mode = CheckMode)
 
+
+GET        /contract-packing                        controllers.ContractPackingController.onPageLoad(mode: Mode = NormalMode)
+POST       /contract-packing                        controllers.ContractPackingController.onSubmit(mode: Mode = NormalMode)
+GET        /change-contract-packing                  controllers.ContractPackingController.onPageLoad(mode: Mode = CheckMode)
+POST       /change-contract-packing                  controllers.ContractPackingController.onSubmit(mode: Mode = CheckMode)
+GET        /how-many-contract-packing                        controllers.HowManyContractPackingController.onPageLoad(mode: Mode = NormalMode)
+POST       /how-many-contract-packing                        controllers.HowManyContractPackingController.onSubmit(mode: Mode = NormalMode)
+GET        /change-how-many-contract-packing                  controllers.HowManyContractPackingController.onPageLoad(mode: Mode = CheckMode)
+POST       /change-how-many-contract-packing                  controllers.HowManyContractPackingController.onSubmit(mode: Mode = CheckMode)
```

Comment:
These need to have the new URL according to your ticket

`/contract-packing-next-12-months`
`/change-contract-packing-next-12-months`